src/routes/safe/components/Transactions/TxsTable/ExpandedTx/TxDescription/CustomDescription.tsx

```diff
@@ -0,0 +1,183 @@
+import { IconText, Text } from '@gnosis.pm/safe-react-components'
+import { makeStyles } from '@material-ui/core/styles'
+import React from 'react'
+import styled from 'styled-components'
+
+import RenderValue from './RenderValues'
```

Comment:
Can we just name it value? It's quite obvious that the component renders something and we don't need to explicitly add it to the name